README.md

```diff
@@ -52,39 +40,67 @@ inventory=your_inventory,..,~/tdp_vars
 enable_plugins = tosit.tdp.inventory,..,your_plugins
 ```
 
-Finally, initialize the database and default variables:
+Install the library:
 
 ```sh
-poetry shell
+# Create and activate a virtual environment
+python -m venv .venv
+source .venv/bin/activate
+# Install the dependencies
+pip install ".[visualization]"
```

Comment:
Using `.` it is necessary to first checkout the source code. You can install the lib with a tarball.
```suggestion
pip install "tdp-lib[visualization]@https://github.com/TOSIT-IO/tdp-lib/tarball/master"
```

Reply:
Good idea, I didn't thought about not having to clone the repository.